packages/studio/src/components/VersionNumber.tsx

```diff
@@ -0,0 +1,8 @@
+import { version } from "../../package.json";
+
+/**
+ * Renders the current version number
+ */
+export default function VersionNumber() {
+  return <p className="text-gray-500">v.{version}</p>;
```

Comment:
can we also get the studio-plugin version?

Reply:
will implement in future! not included in this PR